app/DTO/User.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\DTO;
+
+use App\Models\User as UserModel;
+
+/**
+ * Data Transfer Object (DTO) to transmit the attributes of a User.
+ *
+ * This is used in the Admin part to manage users.
+ */
+class User extends ArrayableDTO
```

Comment:
Why do we need this DTO at all? It does not add anything to a user model which is not already part of a user model. This class seems to be redundant.

Reply:
I updated the comment with the rationale and renamed the DTO to reflect more what it is used for.

If we return directly the User (e.g. for sharing), we are given only the following attributes:
 - id
 - username
 - email
 - has_token

This is because we do not send more than the front-end needs to know.
This DTO aims to provide the attributes of interest in the case of User management.